index.html

```diff
@@ -1090,186 +1090,179 @@ <h1>
 
   <ul class="housing-list">
 
-    <li class="housing  wow animated fadeInLeft icpc" data-wow-delay=".1s">
+    <li class="housing handover wow animated fadeInLeft" data-wow-delay=".1s" style="background-size: 55% auto; background-repeat: repeat;" >
       <div class="housing-holder">
 
         <div class="description">
           <div class="price-and-rating">
-            <div class="price">9<sup>August</sup></div>
+            <div class="price">26<sup>July</sup></div>
             <!--div class="rating"><span class="icon icon-star"></span><span class="icon icon-star"></span><span class="icon icon-star"></span><span class="icon icon-star"></span></div-->
           </div>
           <div class="description-inner">
-            <h2>ICPC MeetUp</h2>
-            <p>Discussed about ACM ICPC and the procedure of participating in it <br>
-                <a href="static/reports/1.pdf" target="_blank">&lt;Report&gt;</a>
+            <h2>Handover Ceremony</h2>
+            <p>The baton was passed from KJSCE Codecell 16-17 to 17-18<br>
+                <a href="#" target="_blank">&lt;Report&gt;</a>
             </p>
           </div>
         </div>
 
       </div>
     </li>
 
-    <li class="housing handover wow animated fadeInLeft" data-wow-delay=".1s" style="background-size: 55% auto; background-repeat: repeat;" >
+    <li class="housing wow animated fadeInLeft event2" data-wow-delay=".1s">
       <div class="housing-holder">
 
         <div class="description">
           <div class="price-and-rating">
-            <div class="price">12<sup>August</sup></div>
+            <div class="price">26<sup>July</sup></div>
```

Comment:
may be we should use this pattern 26 <sup>th<sup> Jul . seems appropriate

Reply:
Do you mean this?
![screenshot from 2018-06-27 02-11-19](https://user-images.githubusercontent.com/38867671/41938034-8f702d1a-79af-11e8-94ed-16e865e0f03e.png)


Or should I decrease the font size of the month a bit like this?
![screenshot from 2018-06-27 02-25-32](https://user-images.githubusercontent.com/38867671/41938712-66af55de-79b1-11e8-91ed-42acaaace46c.png)


Tell me so I can apply one of these changes to all dates. :v:

Reply:
Former Looks better  . emphasis on [dot] at the end . @nurdtechie98 did you mean that ?

Reply:
@akshay-99 yes, go ahead with the first option

Reply:
Except for the date format, everything seems fine.
Also, that seems to be the wrong picture for BE workshop.
We'll keep you posted about the reports and images for the rest of the stuff.

Reply:
okay :+1:

Reply:
@Freelancer-98 we don't have any picture of that workshop so this works.